.checkpatch.conf

```diff
@@ -24,4 +24,5 @@
 --ignore UNDOCUMENTED_DT_STRING
 --ignore DT_SPLIT_BINDING_PATCH
 --ignore DT_SCHEMA_BINDING_PATCH
+--ignore NEW_TYPEDEFS
```

Comment:
No.  This doesn't belong in this PR.

Reply:
Without this line I can not add the div_t and ldiv_t as typeddef structs because of the ci checks.
as far as I saw it should be typedefs according to the standard and use examples i saw online.
Am I right? if yes, you want me to open another pr only for this?

Reply:
No.  In this case the CI checks are wrong and must be ignored.

Reply:
How can I ignore it?

Reply:
Accept that that part of the compliance check will fail.  It's the responsibility of the reviewers to understand that's not your problem to solve.